src/languages/bnf.js

```diff
@@ -0,0 +1,29 @@
+/*
+ Language: Backus–Naur Form
+ Author: Oleg Efimov <efimovov@gmail.com>
+ */
+
+function(hljs){
+  return {
+    contains: [
+      // Attribute
+      {
+        className: 'attr',
+        begin: /</,
+        end: />/
+      },
```

Comment:
This will highlight those both on the left and on the right side of `::=`. How about defining two sides (like in 'ini.js' for example) and then only highlight `<..>` as attrs on the left and everything else on the right. This will be much more robust for detection, too.